HW8/Nudyk.V/password.py

```diff
@@ -0,0 +1,24 @@
+import re
+
+
+def password(user_input):
+    ''' Checks if the password is valid
+    '''
+
+    if  re.findall('[a-z]', user_input) == [] or\
+    re.findall('[A-Z]', user_input) == [] or\
+    re.findall('[0-9]', user_input) == [] or\
+    re.findall('[$,#,@]', user_input) == [] or\
+    len(user_input) < 6 or\
+    len(user_input) > 16:
+        return 'Bad password, pal'
+    else:
+        return 'Very good password, hacker'
+
+
+
+print(password('a'))
+print(password('aZ'))
+print(password('aZ1'))
+print(password('aZ1@'))
+print(password('aZ1@aZ1@'))
```

Comment:
here is another way

import re

password = input("Enter your password:\n")

if re.match("^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[$#@])(?=.{6,16}$).*", password):
    print("Nice pass! Psss, I won't tell anyone ;)")
else:
    print("Password is not valid. Check if you have:"
          """
    At least 1 letter between[a-z] and 1 letter between[A-Z].
    At least 1 number between[0-9].
    At least 1 character from [$#@].
    Minimum length 6 characters.
    Maximum length 16 characters.
          """
           )